src/components/admin/InventoryManager.tsx

```diff
@@ -279,47 +323,136 @@ export default function InventoryManager({
             >
                 <h2 className="text-lg sm:text-xl lg:text-2xl font-bold mb-4 sm:mb-6 text-white uppercase flex items-center gap-2">
                     <CalendarDays size={18} className="sm:w-6 sm:h-6" />
-                    INVENTORY LIST ({inventory.length})
+                    INVENTORY LIST ({getFilteredAndSortedInventory().length} of {inventory.length})
```

Comment:
When filtering our total also decreases. So use 

```suggestion
                    INVENTORY LIST ({inventory.length})
```